docs/how-tos/integrating-with-webflow.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 7
+---
+
+# Integrating Eppo with Webflow
+
+
+In this tutorial, you will learn how how run landing page tests in Webflow after minimal  engineering setup. At a high level, you will add a client side version of Eppo’s Javascript SDK as a script in your Webflow workspace that will:
+
+- initialize the Eppo JS client
+- handle experiment assignment logic
+- dynamically update pre-defined content variables such as text and image URLs
+
+## Prerequisites:
+
+- Eppo Account
+- Webflow account
+- Client-side analytics platform like Segment, Amplitude, Adobe, Mixpanel, etc to capture Eppo assignment events and send them to your data warehouse for analysis.
+
+## Setting up Eppo
+
+Set up an feature flag in Eppo — for step by step instructions, follow our [quickstart](/feature-flags/). We will be using JSON flag type to fill in the content on our Webflow landing page. For the sake of this tutorial, make sure to include the following keys for both variations:
```

Comment:
❤️  Love that we are referencing other docs!

Technically, what you have below are "values" of the variation (vs. "keys") So we may want to use that term to match our UI. We may want to say they need to use JSON type.

<img width="828" alt="image" src="https://github.com/Eppo-exp/eppo-docs/assets/417605/98091b50-0aa9-4426-8e6a-6c37a2981451">

Update: I see now you are saying they just need the keys in the example JSON. It's not exactly clear so maybe say something like "For the sake of this tutorial, make JSON-typed variations that have the same keys as below:"